service/src/main/resources/db/migration/V142__message_attachments.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE attachment
```

Comment:
Indexes? Otherwise getting attachments for a message draft or content requires going through all attachments in the database.

```
CREATE INDEX idx$attachment_message_content ON attachment (message_content_id);
CREATE INDEX idx$attachment_message_draft ON attachment (message_draft_id);
```

Reply:
Jesus christ. I had these in my mind when I started the conversation about index naming on Friday... Good catch! Added one for `income_statement_id` too.